File: Modul 4/Python/PRAK405-2410817120018-ArindaDitaPratiwi.py
a, b = map(int,input().split())
total=0
for i in range(1, a + 1):
    hasil = 0
    for j in range(i, 0, -1):
        hasil += j*b
        print(f"({j} * {b})", end = "")
        if j > 1 :
            print(" + ", end = "")
    print(f" = {hasil}")
    total += hasil
print(total)

File: Modul 4/C/PRAK405-2410817120018-ArindaDitaPratiwi.c
#include <stdio.h>
int main(){
    int a, b;
    int total = 0;
    scanf("%d %d", &a, &b);
    for(int i = 1; i <= a; i++){int hasil = 0;
        for(int j = i; j >= 1; j--){hasil += j * b;
            printf("(%d*%d)", j, b);
            if(j > 1){printf("+");}}
    printf("=%d\n", hasil);
    total += hasil;}
    printf("%d\n", total);
    return 0;}